ts/src/flexible-event/privacy.ts

```diff
@@ -159,10 +171,39 @@ export function maxInformationGain(numStates: number, epsilon: number): number {
   )
 }
 
+export function informationGainWithAttributionScopes(
+  numStates: number,
+  attributionScopeLimit: number,
+  maxEventStates: number
+): number {
+  const totalNumStates = numStates + maxEventStates * attributionScopeLimit
```

Comment:
Should this be attributionScopeLimit - 1?

Reply:
Nice catch. Done.